src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs

```diff
@@ -149,6 +149,7 @@ public void TestReflectInitializationViaSubscription3()
         /// Test request cancellation: reflection hook does not throw
         /// </summary>
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19286")]
```

Comment:
Please re-enable the test, it should be fixed with #19333 and #19280 is updated.